.travis.yml

```diff
@@ -1,25 +1,47 @@
-language: node_js
-node_js: node
-addons:
-  chrome: stable
-script:
+language: node_js
+node_js: node
+addons:
+  chrome: stable
+script:
+- npm run build
```

Comment:
When generating the goldens (from the Travis dropdown), we'll need to remember to also run this since this block of the Travis config is effectively replaced when we go `script: npm run test:diff:golden ...`.

Reply:
Yep good point.